module/zcommon/zfeature_common.c

```diff
@@ -598,6 +598,18 @@ zpool_feature_init(void)
 	zfeature_register(SPA_FEATURE_DRAID,
 	    "org.openzfs:draid", "draid", "Support for distributed spare RAID",
 	    ZFEATURE_FLAG_MOS, ZFEATURE_TYPE_BOOLEAN, NULL);
+
+	{
+	static const spa_feature_t zilsaxattr_deps[] = {
+		SPA_FEATURE_EXTENSIBLE_DATASET,
+		SPA_FEATURE_NONE
+	};
+	zfeature_register(SPA_FEATURE_ZILSAXATTR,
+	    "org.zfsonlinux:zilsaxattr", "zilsaxattr",
```

Comment:
```suggestion
	    "org.openzfs:zilsaxattr", "zilsaxattr",
```

Reply:
Please add the new feature to the zpool-features manpage

Reply:
Ok

Reply:
Done